Formula/l/lbzip2.rb

```diff
@@ -39,7 +39,7 @@ def install
 
   test do
     touch "fish"
-    system "#{bin}/lbzip2", "fish"
```

Comment:
missed l43

Reply:
Yeah that's because it only checks "#{bin}/\<formula-name\>". If we decide we want more we can add it later.